phpugms_greeting_2017-10/index.html

```diff
@@ -0,0 +1,200 @@
+<!doctype html>
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+
+        <title>PHP Usergroup Münster</title>
+
+        <meta name="description" content="PHP Usergroup Münster">
+
+        <meta name="apple-mobile-web-app-capable" content="yes" />
+        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
+
+        <link rel="stylesheet" href="../reveal.js/css/reveal.css">
+        <link rel="stylesheet" href="../theme/theme.css" id="theme">
+
+        <!-- Printing and PDF exports -->
+        <script>
+            var link = document.createElement( 'link' );
+            link.rel = 'stylesheet';
+            link.type = 'text/css';
+            link.href = window.location.search.match( /print-pdf/gi ) ? '../reveal.js/css/print/pdf.css' : '../reveal.js/css/print/paper.css';
+            document.getElementsByTagName( 'head' )[0].appendChild( link );
+        </script>
+
+        <style>
+            .reveal footer {
+                background-color: #7c83b7;
+            }
+
+            .reveal a {
+                color: #50587c;
+            }
+
+
+            .reveal h1:after, .reveal h2:after {
+                background: #50587c;
+            }
+        </style>
+    </head>
+
+    <body class="light">
+        <div class="reveal">
+            <div class="slides">
+
+<section>
+    <h2>
+        <img alt="PHP Usergroup Münster" src="../assets/phpugms-logo.svg" class="noborder" style="width: 300px">
+    </h2>
+
+    <p>
+        <a href="https://www.meetup.com/phpugms">meetup.com/phpugms</a>
+        <br/>
+        <a href="https://twitter.com/phpugms">@phpugms</a>
+        <br/>
+        Wednesday, October 17, 2017
+    <p>
+</section>
+
+                <section>
+                    <h1>Wifi</h1>
+                    <p>SSID: GS Event<br>Password: gruenspar4events</p>
+                </section>
+
+                <section>
+                    <h1>Restroom </h1>
+                </section>
+
+                <section>
+                    <h1>Drinks</h1>
+                </section>
+
+<section>
+    <h2>Thank you sponsors!</h2>
+</section>
+
+
+
+<section>
+    <img src="../assets/gruenspar_logo.jpg" class="noborder" />
+    <p>Thanks for Drinks, Wifi and Hosting</p>
+    <p>Looking for a Job? Ask the team!</p>
+</section>
+                
+
+<section>
+    <h1>Upcoming Events</h1>
+</section>
+
+
+
+                <section>
+                    <img src="img/swhack.png">
```

Comment:
Nice. Are you attending btw?

Reply:
Its marked in my calendar, but i still not have signed up ^^